File: JS/Robo.test.js
/**
 * @version JAVASCRIPT
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ node NomeDaClasse.test
 */
const TEST = require('tape')
const OBJ = require('./Robo.js')

TEST('Starting Robo test...', (t) => {
    //Test 1
    t.assert(OBJ.originPosition("WS") === true, "Expect: true")
    //Test 2
    t.assert(OBJ.originPosition("AA") === false, "Expect: false")

    t.end()
})

File: JAVA/RoboTest.java
/** 
 * @version JAVA
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ mvn clean test -Dtest=your.package.TestClassName
 */
package JAVA;

import static org.junit.Assert.assertEquals;
import org.junit.Test;

public class RoboTest {

    @Test
	public void testRobo() throws Exception {
		//Test 1
		boolean want = true;
		boolean got = Robo.originPosition("WS");
		assertEquals(want, got);
		//Test 2
		want = false;
		got = Robo.originPosition("AA");
		assertEquals(want, got);
	}
}
